tools/kafka_sasl_ssl/generate_ssl_CA/ssl_one_way.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# The example: bash ssl_one_way.sh kafka-0.tigergraph.com ~/SSL_one_way tiger123
+# The example password: tiger123
+# The example server host name: kafka-0.tigergraph.com
+# The certificate generation path: ~/SSL_one_way
+
+if [ $# -eq 3 ]; then
+  server_hostname=$1
+  generate_root=$2
+  pass=$3
+else
+  echo "Error in parameter. Please check."
+  echo "e.g. bash ssl_one_way.sh server_hostname generate_root password"
+  exit 1
+fi
+
+env_prepare(){
+  # install java
+  if ! which java > /dev/null 2>&1; then
+    echo "start install openjdk."
+    sudo yum update -y > /dev/null 2>&1
+    sudo yum install -y java-1.8.0-openjdk > /dev/null 2>&1
```

Comment:
please add option to let user choose to install JDK and other tools or not. If they don't allow to install, quit and report errors.